mr/src/main/java/org/elasticsearch/hadoop/rest/ErrorExtractor.java

```diff
@@ -16,49 +17,71 @@
     public ErrorExtractor(EsMajorVersion internalVersion) {
         this.internalVersion = internalVersion;
     }
+    
+    @SuppressWarnings("rawtypes")
+	public EsHadoopException extractErrorWithCause(Map m) {
```

Comment:
Can you add a real basic unit test for this new code? I know that there was no unit test before, so it doesn't have to be extravagant or anything. Just something simple.

Reply:
Just added an initial suit of tests for ErrorExtractor